src/EditorFeatures/CSharp/SplitComment/SplitCommentCommandHandler.CommentSplitter.cs

```diff
@@ -0,0 +1,140 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Threading;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Formatting;
+using Microsoft.CodeAnalysis.Indentation;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+using Microsoft.CodeAnalysis.Text;
+using static Microsoft.CodeAnalysis.Formatting.FormattingOptions;
+
+namespace Microsoft.CodeAnalysis.Editor.CSharp.SplitComment
+{
+    internal partial class SplitCommentCommandHandler
+    {
+        private class CommentSplitter
+        {
+            protected static readonly SyntaxAnnotation RightNodeAnnotation = new SyntaxAnnotation();
```

Comment:
1. why are these fields protected?
2. it would be worthwhile to doc this field.

Reply:
They can be made private now. They were protected back when I had multiple implementations of the Comment Splitter.

I’ll make them private and add documentation.

Reply:
I've removed it as it was no longer required.